cluster/juju/layers/kubernetes-worker/templates/ingress-replication-controller.yaml

```diff
@@ -45,3 +45,4 @@ spec:
         args:
         - /nginx-ingress-controller
         - --default-backend-service=$(POD_NAMESPACE)/default-http-backend
+        - --nginx-configmap=$(POD_NAMESPACE)/nginx-load-balancer-conf
```

Comment:
Easy : you just create a ConfigMap object. For example :
```
apiVersion: v1
data:
  body-size: 1024m
kind: ConfigMap
metadata:
  name: nginx-load-balancer-conf
```

You can see the list of all the options you can tweak at https://github.com/kubernetes/contrib/blob/master/ingress/controllers/nginx/configuration.md#allowed-parameters-in-configuration-configmap

Using the ConfigMap allows to deploy a "generic" ingress controller that anyone can tweak to their own needs later on.